ntbs-service/Models/Notification.cs

```diff
@@ -227,5 +233,14 @@ private string CreateTimePeriodsString(RiskFactorDetails riskFactor)
             }
             return string.Join(", ", timeStrings);
         }
+
+        private int GetAgeAtTimeOfNotification()
+        {
+            if (NotificationDate == null || PatientDetails?.Dob == null) 
+            {
+                return 0;
+            }
+            return (int) Math.Floor(((DateTime) NotificationDate - (DateTime) PatientDetails.Dob).Days / 365.2425);
+        }
```

Comment:
Do we potentially want this to return a nullable int (and an empty string for age at notification) rather than a 0?
Not sure where I stand on this, just thought I would ask

Reply:
I also questioned to myself whether the implementation of age is a little crude (can report the wrong age for a day in the year).
But if this was acceptable in the old system, I don't see it being an issue here - and a quick google doesn't have anything super clean I would drastically prefer to this.

Reply:
Yes, should be null, will change it.
Their preference was to use this method so that it is consistent with their system